pkg/validatingwebhook/blueprint_handler.go

```diff
@@ -3,13 +3,13 @@ package validatingwebhook
 import (
 	"context"
 	"fmt"
-	"net/http"
 
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
 	kanister "github.com/kanisterio/kanister/pkg"
 	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
 	"github.com/kanisterio/kanister/pkg/blueprint/validate"
+	"net/http"
```

Comment:
This change is not required. `net/http` should be moved back to the list above

Reply:
Done